src/Components/App/App.tsx

```diff
@@ -40,10 +42,31 @@ class App extends React.Component<Props, State> {
       <main className="App">
         <Navbar />
         {this.state.error && <h2>{this.state.error}</h2>}
-        {!this.state.list.length
-          ? !this.state.error && <h2>Loading...</h2>
-          : <List list={this.state.list} />
-        }
+        <Switch>
+
+          <Route
+            exact path='/'
+            render={ () => {
+              return(
+                !this.state.list.length
+                  ? !this.state.error && <h2>Loading...</h2>
+                  : <List list={this.state.list} />
+              )
+            }}
+          />
+
+          <Route
+            exact path='/bookshelf/:queryName'
+            render={ ({ match }) => {
+              const { queryName } = match.params
+              return (
+                <Bookshelf queryID={queryName}/>
+              )
+            }}
+          />
+
+
```

Comment:
Good job implementing router and getting things setup. I'm not sure why it still shows the current page when we add gibberish to the end of the URL. In the past, if I did that it would render the loading or error screen instead. I'm sure we can figure this out.